src/java/org/apache/cassandra/cql3/statements/schema/AlterTypeStatement.java

```diff
@@ -193,7 +193,7 @@ private AlterField(String keyspaceName, String typeName)
 
         UserType apply(KeyspaceMetadata keyspace, UserType userType)
         {
-            throw ire("Alterting field types is no longer supported");
```

Comment:
While we are on this, can we also take care of the is/are issue, please

Reply:
I think 'is' is correct?

Reply:
Leave it as is